crates/wasi-preview1-component-adapter/src/lib.rs

```diff
@@ -36,7 +36,7 @@ pub mod bindings {
         raw_strings,
         // The generated definition of command will pull in std, so we are defining it
         // manually below instead
-        skip: ["run", "get-directories", "get-environment"],
+        skip: ["run", "get-environment", "poll-oneoff"],
```

Comment:
Could you add a comment about why poll-oneoff is skip here, and has manual bindings below instead?

Reply:
https://github.com/bytecodealliance/wasmtime/pull/6498/commits/d76f4f496954ecfa89e98bc9dc239dae8c72a9ce